app/models/school.rb

```diff
@@ -87,6 +87,7 @@ class School < ApplicationRecord
 
   scope :placements_service, -> { where placements_service: true }
   scope :claims_service, -> { where claims_service: true }
+  scope :order_by_name, -> { order(:name) }
```

Comment:
We should provide an explicit direction to our `order` calls to avoid ambiguity.

This is _potentially_ confusing as `order("name")` (String) uses the database default, but `order(:name)` (Symbol) uses `asc`: https://github.com/rails/rails/blob/36b7c1df9b2d66fd59af1f88ad74426695aa756b/activerecord/lib/active_record/relation/query_methods.rb#L1992-L2012

```suggestion
  scope :order_by_name, -> { order(name: :asc) }
```